views/components/form-fields-next/macros.njk

```diff
@@ -495,6 +495,34 @@
     </li>
 {% endmacro %}
 
+{% macro inputFile(field) %}
+
+    {% if field.value and field.displayValue %}
+        <div class="existing-data u-margin-bottom">
+            <h3>Existing file</h3>
+            <p>{{ field.displayValue }}</p>
+            <p class="u-no-margin">
+                <a href="../download/{{ field.name }}/{{ field.value.filename }}">Download this file</a>
```

Comment:
We might need to define this download route somewhere on the model? This relative path might not make sense depending on where/how this macro us rendered. Doesn't need to be solved now though.